code/game/objects/items/storage/belt.dm

```diff
@@ -11,7 +11,7 @@
 	attack_verb = list("whipped", "lashed", "disciplined")
 	w_class = WEIGHT_CLASS_BULKY
 	allow_drawing_method = TRUE
-
+	refill_types = list(/obj/item/storage/box/visual/magazine)
```

Comment:
Does this need to be on /belt instead of /belt/marine?

Reply:
I had it on belt/marine then moved it up, I can't remember why anymore.
I guess I'll just move it back down.

Reply:
Oh yeah, now I remember.
Turns out, shotgun shell load rig is /obj/item/storage/belt/shotgun
And not the expected /obj/item/storage/belt/marine/shotgun